common/src/main/scala/explore/implicits.scala

```diff
@@ -3,12 +3,42 @@
 
 package explore
 
+import scala.annotation.unused
```

Comment:
Oddly, this seems unused

Reply:
It's used here: https://github.com/gemini-hlsw/explore/blob/ea8ae6a7cb0ab6cf517283bf33ee868073a48800/common/src/main/scala/explore/implicits.scala#L21